--- .\meson.build.orig	Fri Dec 30 03:04:49 2022
+++ .\meson.build	Fri Dec 30 03:03:28 2022
@@ -96,7 +96,7 @@ endif
 
 api = '-DIMGUI_API=__attribute__((visibility("default")))'
 if host_machine.system() == 'windows'
-  api = '-DIMGUI_API=@0@'.format(get_option('buildtype') != 'static' ? '__declspec(dllexport)' : '""')
+  api = '-DIMGUI_API=@0@'.format(get_option('default_library') != 'static' ? '__declspec(dllexport)' : '')
 endif
 
 imgui = library(
@@ -106,10 +106,11 @@ imgui = library(
   gnu_symbol_visibility: 'hidden',
   dependencies: dependencies,
   include_directories: include_dirs,
+  install: true,
 )
 
 if host_machine.system() == 'windows'
-  api = '-DIMGUI_API=@0@'.format(get_option('buildtype') != 'static' ? '__declspec(dllimport)' : '""')
+  api = '-DIMGUI_API=@0@'.format(get_option('default_library') != 'static' ? '__declspec(dllimport)' : '')
 endif
 
 imgui_dep = declare_dependency(
